src/Facebook/FacebookRequest.php

```diff
@@ -102,7 +102,8 @@ public function __construct(FacebookApp $app = null, $accessToken = null, $metho
         $this->setEndpoint($endpoint);
         $this->setParams($params);
         $this->setETag($eTag);
-        $this->graphVersion = $graphVersion ?: Facebook::DEFAULT_GRAPH_VERSION;
+        // @TODO Remove this fall-back
+        $this->graphVersion = $graphVersion ?: 'v2.7';
```

Comment:
I think the way to handle this case is to allow null graph version in requests. This parameter is here to override the version given in the client, so it can be null, we just have to take care of this value when sending the request, falling back to the client version when this is null :)